core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp

```diff
@@ -22,11 +22,11 @@
     },
     {
       "name": "LDAP_URL",
-      "referenceConfiguration": "hive.server2.authentication.ldap.url"
+      "referenceConfiguration": "ranger.usersync.ldap.url"
```

Comment:
No they needs to be deleted, they do not make sense at all.  I have raised this issue: https://hortonworks.jira.com/browse/BUG-78138